heron/ckptmgr/src/java/BUILD

```diff
@@ -0,0 +1,40 @@
+licenses(["notice"])
+
+package(default_visibility = ["//visibility:public"])
+
+load("/tools/rules/jarjar_rules", "jarjar_binary")
+
+java_library(
+    name = "ckptmgr-java",
+    srcs = glob(
+        ["**/*.java"],
+        exclude = ["**/CheckpointManager.java"],
```

Comment:
`CheckpointManager.java` is the entry point of the checkpoint management, which includes the `main` method.

since `ckptmgr-java` is a `java_library`, the entry point class is not needed at the library level. On the other hand, `ckptmgr-unshaded` is a `java_binary`, it needs the `CheckpointManager.java`.